nucypher/cli/painting.py

```diff
@@ -886,42 +886,51 @@ def paint_worklock_status(emitter, registry: BaseContractRegistry):
 
     now = maya.now()
-    bidding_remaining = bidding_end - now if bidding_end > now else timedelta()
-    cancellation_remaining = cancellation_end - now if cancellation_end > now else timedelta()
+    bidding_remaining = bidding_end - now if bidding_end > now else 'Closed'
+    cancellation_remaining = cancellation_end - now if cancellation_end > now else "Closed"
 
+    cancellation_open = bidding_start <= now <= cancellation_end
+    bidding_open = bidding_start <= now <= bidding_end
 
     payload = f"""
-
 Time
-======================================================
-
-Bidding open .................. {'Yes' if bidding_open else 'No'} 
-Bidding Start Date ................... {bidding_start}
-Bidding End Date ..................... {bidding_end}
-Bidding Duration ..................... {bidding_duration}
-Bidding Time Remaining ............... {bidding_remaining} 
+══════════════════════════════════════════════════════
 
+Contribution ({'Open' if bidding_open else 'Closed'})
 ------------------------------------------------------
+Claims Available ...... {'Yes' if worklock_agent.is_claiming_available() else 'No'}
+Start Date ............ {bidding_start}
+End Date .............. {bidding_end}
+Duration .............. {bidding_duration}
+Time Remaining ........ {bidding_remaining} 
 
-Claiming open .................. {'Yes' if worklock_agent.is_claiming_available() else 'No'}
-Cancellation Window End Date ......... {cancellation_end}
-Cancellation Window Duration ......... {cancellation_duration}
-Cancellation Window Time Remaining ... {cancellation_remaining}
+Cancellation ({'Open' if cancellation_open else 'Closed'})
+------------------------------------------------------
+End Date .............. {cancellation_end}
+Duration .............. {cancellation_duration}
+Time Remaining ........ {cancellation_remaining}
+ 
  
-
 Economics
-======================================================        
-Min allowed bid ....... {prettify_eth_amount(worklock_agent.minimum_allowed_bid)}
-ETH Pool .............. {prettify_eth_amount(blockchain.client.get_balance(worklock_agent.contract_address))}
-ETH Supply ............ {prettify_eth_amount(worklock_agent.get_eth_supply())}
-Bonus ETH Supply ...... {prettify_eth_amount(worklock_agent.get_bonus_eth_supply())}
+══════════════════════════════════════════════════════
 
-Number of bidders...... {worklock_agent.get_bidders_population()}
+Participation
+------------------------------------------------------
 Lot Size .............. {NU.from_nunits(worklock_agent.lot_value)} 
-Bonus Lot Size ........ {NU.from_nunits(worklock_agent.get_bonus_lot_value())} 
+Min. Allowed Bid ...... {prettify_eth_amount(worklock_agent.minimum_allowed_bid)}
+Participants .......... {worklock_agent.get_bidders_population()}
+ETH Pool .............. {prettify_eth_amount(blockchain.client.get_balance(worklock_agent.contract_address))}
+ETH Supply ............ {prettify_eth_amount(worklock_agent.get_eth_supply())}
```

Comment:
What's the difference between ETH pool and ETH supply?

Reply:
`ETH Pool` shows how many eth in the current moment, so after starting refund `ETH Pool` will deacrease and `ETH Supply` will be the same